docs/pywbemcli/cmdshelp.rst

```diff
@@ -867,7 +867,7 @@ Help text for ``pywbemcli help`` (see :ref:`help command`):
 
 ::
 
-    Usage: pywbemcli help [OPTIONS]
+    Usage: pywbemcli [OPTIONS] help
```

Comment:
Can `OPTIONS` be renamed to `GENERAL-OPTIONS` ?

Reply:
The change addresses the second comment but not the first. That's actually ok because we don't show it in other cases that have only the --help option, either. DONE.